sites/fast-website/webpack.common.js

```diff
@@ -37,42 +36,40 @@ module.exports = {
             {
                 test: /\.css$/,
                 exclude: /node_modules/,
-                use: [
-                    {
-                        loader: MiniCssExtractPlugin.loader,
-                        options: {
-                            esModule: false,
-                        },
-                    },
-                    {
-                        loader: "css-loader",
-                    },
-                ],
```

Comment:
Using the html-bundler-webpack-plugin, the MiniCssExtractPlugin.loader is not need more.
This plugin can extract CSS.